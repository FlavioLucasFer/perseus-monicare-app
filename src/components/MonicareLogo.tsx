import React from 'react';
import { Image, ImageProps } from 'react-native';

interface MonicareLogoProps extends Omit<ImageProps, 'source'> {};

const MonicareLogo: React.FC<MonicareLogoProps> = props => (
	<Image 
		{...props}
		style={[{ resizeMode: 'contain', width: '100%' }, props.style]}
		source={require('images/monicare-logo.png')}
	/>
);

export type {
	MonicareLogoProps,
};
export default MonicareLogo;
