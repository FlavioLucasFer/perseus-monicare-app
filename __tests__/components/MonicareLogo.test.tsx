import React from 'react';
import { shallow } from 'enzyme';

import MonicareLogo from 'components/MonicareLogo';

describe('MonicareLogo component', () => {
	it('Should render properly', () => {
		const wrapper = shallow(
			<MonicareLogo />
		);
		expect(wrapper.debug()).toMatchSnapshot();
	});
});
