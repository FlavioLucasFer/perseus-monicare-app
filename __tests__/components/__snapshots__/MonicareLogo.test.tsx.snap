// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MonicareLogo component Should render properly 1`] = `"<Image style={{...}} source={{...}} />"`;
